### DISCUSSION TITLE
Making diagrams DBMS specific

### DISCUSSION THREAD

**Discussion by 1ilit** [drawdb-io/drawdb#38] [Polls]
Hey everyone! 

As you might have noticed drawDB supports a limited selection of datatypes which get converted to their closest matches in DBMS-s that don't support them.

Clearly this can become inconvenient if you need datatypes outside of the supported ones. To remedy this I've been thinking of making diagrams DBMS specific. This will require the user to pick a DBMS before entering the editor after which all the supported datatypes will be available. This, however, will only allow exporting to the SQL flavor that was selected in advance.

On the other hand, we can have a limited set(more types can be added but certainly not all) that get translated to their closest matches in all available SQL flavors. 

I want to hear your thoughts and opinions on how to proceed.

Thank you:)

**Comment by ghost** [comment]
Would be better
